test/Container/ResponseFactoryFactoryWithoutDiactorosTest.php

```diff
@@ -1,66 +0,0 @@
-<?php
-
-/**
- * @see       https://github.com/mezzio/mezzio for the canonical source repository
- * @copyright https://github.com/mezzio/mezzio/blob/master/COPYRIGHT.md
- * @license   https://github.com/mezzio/mezzio/blob/master/LICENSE.md New BSD License
- */
-
-declare(strict_types=1);
-
-namespace MezzioTest\Container;
-
-use Mezzio\Container\Exception\InvalidServiceException;
-use Mezzio\Container\ResponseFactoryFactory;
-use PHPUnit\Framework\TestCase;
-use Prophecy\Prophecy\ObjectProphecy;
-use Psr\Container\ContainerInterface;
-
-use function class_exists;
-use function spl_autoload_functions;
-use function spl_autoload_register;
-use function spl_autoload_unregister;
-
-class ResponseFactoryFactoryWithoutDiactorosTest extends TestCase
```

Comment:
What was the reason to remove these tests? It would be nice to mention that on the commit message.